main.cpp

```diff
@@ -73,29 +73,36 @@ struct <#structName1#>                                //4
 
 struct U
 {
-    float <#name1#> { 0 }, <#name2#> { 0 };
-    <#returnType#> <#memberFunction#>(<#type name#>* <#updatedValue#>)      //12
+    float name1 { 30 }, name2 { 20 };
+    float memberFunction(float* updatedValue)      //12
     {
-        
+        std::cout << "U's name1 value: " << name1 << std::endl;
+        name1 = *updatedValue;
+        std::cout << "U's name1 updated value: " << name1 << std::endl;
```

Comment:
What do we always check before using pointers? 
- [ ] revise

Reply:
Always check to make sure your pointer is not NULL before using it!!!! :)